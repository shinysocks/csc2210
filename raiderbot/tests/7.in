+--------------------+  
|    **# ########  | |
|  # **T#*   T*T   --|
| #  **T=*TT.*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
19 0   
www
swww
esswwn
wwwwns
wwwn
wwn
esseeee  
ssws   
