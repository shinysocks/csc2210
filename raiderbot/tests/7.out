Initial state: Robot at 19, 0 (0 gold)
+--------------------+
|    **# ########  |R|
|  # **T#*   T*T   --|
| #  **T=*TT.*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
========================================
Executing www
Robot at 16, 0 (0 gold)
+--------------------+
|    **# ########R | |
|  # **T#*   T*T   --|
| #  **T=*TT.*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing swww
Robot now owes the bank 1 pieces.
Robot at 13, 1 (0 gold)
+--------------------+
|    **# ########  | |
|  # **T#*   TRT   --|
| #  **T=*TT.*#*     |
| #     #*    *      |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing esswwn
Robot now owes the bank 1 pieces.
Robot at 12, 2 (2 gold)
+--------------------+
|    **# ########  | |
|  # **T#*   T T   --|
| #  **T=*TT.R#      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwwwns
Robot at 8, 2 (2 gold)
+--------------------+
|    **# ########  | |
|  # **T#    T T   --|
| #  **T=RTT. #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwwn
Robot at 5, 1 (3 gold)
+--------------------+
|    **# ########  | |
|  # *RT#    T T   --|
| #  * T= TT. #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing wwn
Robot at 3, 0 (4 gold)
+--------------------+
|   R**# ########  | |
|  #   T#    T T   --|
| #  * T= TT. #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing esseeee
Robot at 8, 2 (5 gold)
+--------------------+
|     *# ########  | |
|  #   T#    T T   --|
| #    T=RTT. #      |
| #     #*           |
|  #   #*    #B#     |
|       *    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
Executing ssws
Robot at 7, 5 (8 gold)
+--------------------+
|     *# ########  | |
|  #   T#    T T   --|
| #    T= TT. #      |
| #     #            |
|  #   #     #B#     |
|       R    #  #####|
|     #####  #    #  |
|    #     ### # #   |
|  ##        S       |
|             ###### |
+--------------------+
========================================
Robot has completed its task.
